avm/res/operational-insights/workspace/data-export/main.bicep

```
@@ -53,3 +54,21 @@ output resourceId string = dataExport.id
 
 @description('The name of the resource group the data export was created in.')
 output resourceGroupName string = resourceGroup().name
+
+// =============== //
+//   Definitions   //
+// =============== //
+
+@export()
+type destinationType = {
```

Just in case you did not know - this works too 😄 

```suggestion
@description('A type for a data export destination.')
type destinationType = {
```

---

If you want to adopt this suggestion, the same comment applies to all other types. If not, nevermind. But I think there'll be a spec eventually 😄

---

Thanks! No, I didn't know it, I will add the descriptions.